site/_docs/themes.md

```diff
@@ -14,7 +14,11 @@ Jekyll has an extensive theme system, which allows you to leverage community-mai
 
 2. Save the changes to your `Gemfile`
 3. Run the command `bundle install` to install the theme
-4. Finally, activate the theme by adding the following to your site's `_config.yml`:
+4. Set your `/css/main.scss` to import your theme partials using the `@import` directive:
```

Comment:
@parkr Can you create a blog-post *detailing* the addition of the assets folder as part of `SCAFFOLD_DIRECTORIES` like how to use this new feature.

Right now, I'm confused if moving `css/main.scss' generated by `jekyll new` can be moved to `/assets` and it'll still be read and processed..